packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1302,6 +1302,18 @@ export const UnknownFilesystemError = {
 	hint: 'It can be caused by many things, eg. missing permissions or a file not existing we attempt to read. Check the error cause for more details.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Your `tsconfig.json` `extends` property is invalid
+ */
+export const TSConfigInvalidExtends = {
+	name: 'TSConfigInvalidExtends',
+	title: 'Your `tsconfig.json` `extends` property is invalid',
+	message: 'Your `tsconfig.json` `extends` property is invalid',
+	hint: 'Check out what change you need to make in the terminal above.',
```

Comment:
```suggestion
	hint: 'Check the terminal output for more detail.',
```
Something like this maybe? If you can guide them more closely to exactly what they should see in the terminal (e.g. "exactly what went wrong", exactly what caused the error) then I'd do that.

If this error is only reached when there's e.g. a typo in one of the two things you have to extend (the astro TS template, and the new file), then the hint could be something like "Check the [`experimental.tsconfig` documentation]() for the proper `extends` configuration"

Reply:
It's showing a diff to help the user, your suggestion lgtm